content/platforms/kubernetes/cluster-recovery.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: Cluster Recovery in Kubernetes
+description: 
+weight: 40
+alwaysopen: false
+categories: ["Platforms"]
+aliases: /rs/concepts/kubernetes/cluster-recovery
+---
+
+## Cluster Recovery
+If the cluster losses a majority of its nodes (e.g. from nodes crashing or network split), the cluster will not responde. Recovering the cluster requires performing the following steps:
+
```

Comment:
Couldn't find one but it would be preferable If we can refer to an article that explains how to detect a failure scenario that **mandates** recovery. We can give an example from `rladmin status` or of a split cluster.
We should also mention that this process can be used to intentionally tear down a cluster and rebuild it - for example for the purpose of recycling nodes, active/cold-passive failover, etc.

Reply:
@amiramm I don't think the other recovery article that we have adds much to this. It also might confuse readers because it is specifically in the context of RS.
I'll add the points about recycling nodes and active/passive failover.

Reply:
Ideally, I would list which of the action in the RS article the operator does automate vs. the ones that still need to be done by the user, which are listed as steps in this article:
Operator automates all of these steps:

1. Recreates a fresh Redis Enterprise cluster
2. Mounts the persistent storage with the recovery files from the original cluster to the nodes of the new cluster
3. Recovers the cluster configuration on the first node in the new cluster
4. Joins the remaining nodes to the new cluster.

Agree though that it may confuse, so I'd leave it at your discretion.

Reply:
Added